syntax = 'proto3';

service regulatoryComplianceMS{
  rpc getLocations(Locations) returns(LocationStatus);
  rpc availableLocations(LocationRequest) returns (LocationReply);
}

message Locations{
   string locations = 1;//receive 5 locations in a single string
}

message LocationStatus{
    string status = 1;//query these locations and return their status (common, condo, corridor, reserve)
}

message LocationRequest{
    string request = 1; //send a string to initiate the service "available locations"
}

message LocationReply{
    string availLocations = 1;//reply with all available locations in a single string
}

//python -m grpc_tools.protoc -I./protos --python_out=./generated --grpc_python_out=./generated ./protos/example.proto